css/style.css

```diff
@@ -1,19 +1,145 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  background-size: contain;
+}
+.background {
+  background-image: url("../img/first-background.jpg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  height: 60vh;
+  position: top;
+}
+.navbar {
+  background-color: white;
+  margin: 0px;
+  padding: 0px;
+  list-style-type: none;
+  text-align: right;
+}
+.nav-item {
+  padding: 0px;
+  margin: 10px;
+  color: gray;
+  display: inline-block;
+}
+.nav-item:hover {
+  color: orange;
+}
+#nav-link1 {
+  color: black;
+}
+.logo {
+  margin: 5px;
+  padding: 0px;
+  width: 30px;
+  float: left;
+}
+nav select {
+  display: none;
+}
+h1 {
+  margin: 0px;
+  color: white;
+  text-align: center;
+  font-weight: normal;
+}
+p {
+  color: white;
+  text-align: center;
+}
+div {
+  color: white;
+  text-align: center;
+  border: 0px;
+}
+button {
+  background-color: rgb(240, 128, 0);
+}
```

Comment:
Might want to add a `padding` property and a `border-radius` property to match the photo.